app/assets/javascripts/discourse/app/components/invite-link-panel.js

```diff
@@ -0,0 +1,122 @@
+import I18n from "I18n";
+import Component from "@ember/component";
+import Group from "discourse/models/group";
+import { alias } from "@ember/object/computed";
+import discourseComputed from "discourse-common/utils/decorators";
+import Invite from "discourse/models/invite";
+
+export default Component.extend({
+  inviteModel: alias("panel.model.inviteModel"),
```

Comment:
would prefer readOnly instead of all the alias here

Reply:
Done.